src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,20 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        Truck truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
+
+        Machine[] array = {truck, bulldozer, excavator};
```

Comment:
```suggestion
        Machine[] machines = {truck, bulldozer, excavator};
```
naming by data type isn't informative